parts/linux/cloud-init/artifacts/cse_cmd.sh

```diff
@@ -67,4 +67,5 @@ SGX_NODE={{GetVariable "sgxNode"}}
 CONFIG_GPU_DRIVER_IF_NEEDED={{GetVariable "configGPUDriverIfNeeded"}}
 ENABLE_GPU_DEVICE_PLUGIN_IF_NEEDED={{GetVariable "enableGPUDevicePluginIfNeeded"}}
 TELEPORTD_PLUGIN_DOWNLOAD_URL={{GetParameter "teleportdPluginURL"}}
+CONTAINERD_VERSION={{GetParameter "containerdVersion"}}
```

Comment:
nit: Rename to `GOAL_CONTAINERD_VERSION`? 
reason: I believe all of these vars set here will be globally scoped so down the line where you pass this variable into installStandloneContainerd as a parameter it's kinda redundant since this var name is being reused everywhere.

Reply:
I had a similar concern, i suggested the caller to be using a different name so that this variable and callers don't get mixed up. @anujmaheshwari1 you can go with either option.